test/messages/ext_request.test.js

```diff
@@ -37,6 +61,7 @@ test('new with args', function (t) {
   t.ok(req);
   t.equal(req.requestName, '1.2.3.4');
   t.equal(req.requestValue, 'test');
+  t.ok(areBuffersEqual(req.requestValueBuffer, new Buffer('test', 'utf8')));
```

Comment:
We should probably be testing the public API too

```suggestion
  t.ok(areBuffersEqual(req.requestValueBuffer, new Buffer('test', 'utf8')));
  t.equal(req.value, 'test');
  t.ok(areBuffersEqual(req.valueBuffer, new Buffer('test', 'utf8')));
```